src/plugins/charts/public/services/colors/colors_palette.test.ts

```diff
@@ -46,6 +46,12 @@ describe('Color Palette', () => {
     colorPalette = createColorPalette(num1);
   });
 
+  function isHexValue(value: string): boolean {
+    // Check if the hex value is valid.
+    const regex = /^#[0-9a-fA-F]{3}|[0-9a-fA-F]{6}$/;
+    return regex.test(value) ? true : false;
```

Comment:
Why have a ternary at all? Why not just `return regex.test(value);`?

Reply:
you're right!

Reply:
This file will be deleted in a follow-up PR https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4398/files